test/spirv-load-store.ll

```diff
@@ -0,0 +1,68 @@
+; Translate SPIR-V friendly OpLoad and OpStore calls
+; RUN: llvm-as %s -o %t.bc
+; RUN: llvm-spirv %t.bc -spirv-text -o %t
+; RUN: FileCheck < %t %s
+; RUN: llvm-spirv %t.bc -o %t.spv
```

Comment:
Maybe add a `spirv-val` step too?

Reply:
I've added it in the original commit, but existence of __spirv_BuiltInGlobalInvocationId results in a failure (if I remember correctly in a known SPIR-V 1.4 entry validation failure). I'll rewrite the test, removing __spirv_BuiltInGlobalInvocationId and adding spirv-val.